packages/functional-tests/pages/login.ts

```diff
@@ -14,7 +14,7 @@ export const selectors = {
   EMAIL_PREFILLED: '#prefillEmail',
   EMAIL_HEADER: '#fxa-enter-email-header',
   ERROR: '.error',
-  LINK_LOST_RECOVERY_KEY: 'a:has-text("Don’t have an account recovery key?")',
+  LINK_LOST_RECOVERY_KEY: '.lost-recovery-key',
```

Comment:
The previous selector didn't actually work, because the string in use for backbone has a straight quote (`Don't`) vs (`Don’t`) in the selector. I considered updating the original string, but it didn't feel worth triggering a re-translation since we're so close to stripping out the backbone flow for reset password anyway. And it seems fine for this to only work for backbone - we should be using a selector in the `resetPasswordReact` page (vs `login`) for this moving forward.